contrib/npm/hurlfmt.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+
+const bin = require("./bin")
+
+bin.run("hurlfmt");
```

Comment:
I wonder if the README should mention that `hurlfmt` is also available, what it does, and how to access it?